### PR TITLE
Page redesign

### PR DESCRIPTION
**Pull request by manutuero**
### Summary

Change the content layout using CSS Grids

### Trello Card
https://trello.com/c/skPKVr2m

### Screenshot
new-layout

### CODE REVIEW

File: scss/tech.scss [hunk 1]
```diff
@@ -1,44 +1,64 @@
+.tech-container {
+  display: grid;
+  grid-template-columns: 200px 200px 200px;
```
Comment (CarlosHWolox) [inline scss/tech.scss]: You can use repeat() css function here

File: scss/tech.scss [hunk 2]
```diff
@@ -1,44 +1,64 @@
...
+.primary-item {
+  grid-row: 1/3;
+  grid-column: 1/2;
+  border-right: 2px solid $green;
+  justify-self: center;
+  padding: 0 10px;
+  height: 100%;
+}
+
+.item-1 {
```
Comment (CarlosHWolox) [inline scss/tech.scss]: Remember that the only tech card that is different is the first one, the other ones use the same 'style'.

File: scss/tech.scss [hunk 3]
```diff
@@ -1,44 +1,64 @@
...
-
-  &:hover::before {
-    top: 10px;
-    left: 10px;
-  }
+  width: 200px;
 
   .tech-name {
     font-size: $text-medium;
+    margin-bottom: 10px;
```
Comment (CarlosHWolox) [inline scss/tech.scss]: you can apply this directly to all .tech-name, .tech-release, .tech-logo

File: scss/tech.scss [hunk 4]
```diff
@@ -1,31 +1,28 @@
...
-  &:hover::before {
-    top: 10px;
-    left: 10px;
+  &:first-child {
+    grid-row: 1/3;
+    grid-column: 1/2;
+    border-right: 2px solid $green;
+    justify-self: center;
+    padding: 0 10px;
+    height: 100%;
```
Comment (CarlosHWolox) [inline scss/tech.scss]: Define a min-height is better than using 100% as if it is shorter it would look bad.

**Review by CarlosHWolox — APPROVED** [review]

File: scss/tech.scss [hunk 5]
```diff
@@ -1,31 +1,28 @@
+.tech-container {
+  display: grid;
+  grid-template-columns: repeat(3, 200px);
```
Comment (SKOLZ) [inline scss/tech.scss]: I'd change this to `grid-template-columns: repeat(3, auto);` since you  already define the 200px in the tech.

**Review by SKOLZ — APPROVED** [review]

**Pull request merged**